vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SampleRegistrationSpreadsheet.java

```diff
@@ -269,22 +292,24 @@ private void prepareCommonSheetTasks() {
     setupCommonDropDownColumns();
   }
 
-  void unlockCellsToRow(int maxRow, int column) {
+  void enableEditableCellsOfColumnUntilRow(int maxRow, int column) {
     Set<Cell> cells = new HashSet<>();
     for (int row = 1; row <= maxRow; row++) {
       if (isCellUnused(this.getCell(row, column))) {
         Cell cell = this.createCell(row, column, "");
         cells.add(cell);
       }
     }
-    defaultStyleAndUnlockCells(cells);
+    defaultStyleAndUnlockEditableCells(cells);
   }
 
-  void defaultStyleAndUnlockCells(Set<Cell> cells) {
+  void defaultStyleAndUnlockEditableCells(Set<Cell> cells) {
```

Comment:
What does defaultStyle mean since the only change i can see is that the cells are not locked?

Reply:
I introduced this name in the validation highlighting PR. It means cells with "normal" text and without any highlighting.